### PR TITLE
Weed out duplicates

### PR DESCRIPTION
**Pull request by DarthHater**
Basically, this will remove duplicate hashes that `hashbrowns` runs into. Copies something similar from `nancy`. Adds a duplicate to the parse testdata to ensure everything still works fine.

### CODE REVIEW

File: cmd/fry.go [hunk 1]
```diff
@@ -108,7 +108,7 @@ func doParseSha1List(config *types.Config) (exitCode int, err error) {
 	}
 
 	log.WithField("path", config.Path).Info("Beginning parsing of file into sha1 type")
-	sha1s, err := parse.ParseSha1File(config.Path)
+	sha1s, err := parse.Sha1File(config.Path)
```
Comment (DarthHater) [inline cmd/fry.go:111]: Suggestion from go, basically `parse.ParseSha1File` is a stutter, this is `parse.Sha1File` which reads perfectly.

Comment (bhamail) [reply cmd/fry.go:111]: Makes sense.

File: parse/testdata/thing.txt [hunk 2]
```diff
@@ -1,2 +1,3 @@
 9987ca4f73d5ea0e534dfbf19238552df4de507e  main.go
 2a72a07fbc9de22308d12a32f7d33504349e63c9  Makefile
+2a72a07fbc9de22308d12a32f7d33504349e63c9  Makefile
```
Comment (bhamail) [inline parse/testdata/thing.txt:3]: This excellent test coverage is making my day!

**Review by bhamail — APPROVED** [review]
+1

**Pull request merged**